detekt.properties

```diff
@@ -1,14 +1,20 @@
 category=External Analysers
 description=Provide detekt rules for Kotlin projects
 homepageUrl=https://github.com/detekt/sonar-kotlin
-archivedVersions=2.0.0,2.2.0
-publicVersions=2.1.0,2.3.0
+archivedVersions=2.0.0,2.2.0,2.3.0
+publicVersions=2.1.0,2.4.0
 
 defaults.mavenGroupId=io.github.detekt
 defaults.mavenArtifactId=sonar-detekt
 
+2.4.0.description=Upgrade to detekt 1.18.1
+2.4.0.sqVersions=[7.9.4,LATEST]
+2.4.0.date=2021-10-31
+2.4.0.changelogUrl=https://github.com/detekt/sonar-kotlin/releases/tag/2.4.0
+2.4.0.downloadUrl=https://github.com/detekt/sonar-kotlin/releases/download/2.4.0/sonar-detekt-2.4.0.jar
```

Comment:
This seems like a really big jar. Remind me, please: it bundles Detekt?

Reply:
Yes that's the case:
https://github.com/detekt/sonar-kotlin/releases

Specifically we bundle the `detekt-cli` which you can find here having the same size: https://github.com/detekt/detekt/releases/tag/v1.19.0